charts/underwriter/templates/configmap.yaml

```
@@ -28,7 +28,7 @@ data:
   # PostgreSQL Database
   POSTGRES_HOST: {{ .Values.underwriter.configmap.POSTGRES_HOST | default "underwriter-postgresql-primary" | quote }}
   POSTGRES_PORT: {{ .Values.underwriter.configmap.POSTGRES_PORT | default "5432" | quote }}
-  POSTGRES_DB: {{ .Values.underwriter.configmap.POSTGRES_DB | default "underwriter" | quote }}
+  POSTGRES_NAME: {{ .Values.underwriter.configmap.POSTGRES_NAME | default "underwriter" | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)